RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -659,6 +421,30 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     float y2 = mds.anchorY()[secondMDIndex];
     float y3 = mds.anchorY()[thirdMDIndex];
 
+    const float rt_InLo = mds.anchorRt()[firstMDIndex];
+    const float rt_InOut = mds.anchorRt()[secondMDIndex];
+    const float sdIn_alpha = __H2F(segments.dPhiChanges()[innerSegmentIndex]);
+
+    const float drt_InSeg = rt_InOut - rt_InLo;
+    
+    const float tl_axis_x = x3 - x1;
+    const float tl_axis_y = y3 - y1;
+
+    betaIn = sdIn_alpha - cms::alpakatools::reducePhiRange(
+                              acc, cms::alpakatools::phi(acc, tl_axis_x, tl_axis_y) - mds.anchorPhi()[firstMDIndex]);
+
+    float drt_tl_axis = alpaka::math::sqrt(acc, tl_axis_x * tl_axis_x + tl_axis_y * tl_axis_y);
+
+    //innerOuterAnchor - innerInnerAnchor
+    float rt_InSeg = alpaka::math::sqrt(acc,
+                                  (mds.anchorX()[secondMDIndex] - mds.anchorX()[firstMDIndex]) *
+                                  (mds.anchorX()[secondMDIndex] - mds.anchorX()[firstMDIndex]) +
+                                  (mds.anchorY()[secondMDIndex] - mds.anchorY()[firstMDIndex]) *
+                                  (mds.anchorY()[secondMDIndex] - mds.anchorY()[firstMDIndex]));
```

Comment:
temporaries were already defined for this
```suggestion
                                  (x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1));
```

Reply:
This is fixed, thank you